spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingAutoConfiguration.java

```diff
@@ -44,6 +44,9 @@
 	@Autowired
 	private DefaultPollerProperties poller;
 
+	@Autowired(required = false)
+	List<Bindable> adapters;
```

Comment:
Good point, will fix. Also, needs to be initialized to an empty list, to avoid passing a null reference.